contents/docs/error-tracking/installation/python.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Python error tracking installation
+---
+
+import { Steps, Step } from 'components/Docs/Steps'
+
+import Tab from "components/Tab"
+import DjangoContextMiddleware from '../../libraries/python/_snippets/django-context-middleware.mdx'
+import PythonFlaskExceptionAutocapture from '../../libraries/python/_snippets/python-flask-exception-autocapture.mdx'
+import PythonFastAPIExceptionAutocapture from '../../libraries/python/_snippets/python-fastapi-exception-autocapture.mdx'
+
+<Steps>
+<Step title="Installing PostHog Python SDK" subtitle="Your goal in this step: Install the PostHog Python SDK." badge="required">
+
+```bash
+pip install posthog
+```
+
+In your app, import the `posthog` library and set your project API key and host **before** making any calls.
+
+```python
+from posthog import Posthog
+
+
+posthog = Posthog('<ph_project_api_key>', host='<ph_client_api_host>')
+```
+
+> **Note:** As a rule of thumb, we do not recommend having API keys in plaintext. Setting it as an environment variable is best.
+
+You can find your project API key and instance address in the [project settings](https://app.posthog.com/project/settings) page in PostHog.
+
+To debug, you can toggle debug mode:
+
+```python
+posthog.debug = True
+```
+
+To make sure no calls happen during tests, you can disable PostHog, like so:
+
+```python
+if settings.TEST:
+    posthog.disabled = True
+```
```

Comment:
I don't fully understand why this is relevant here. It's a bit useful but probably less than everything else on the page. I'd remove tbh, especially since you verify being going in-app.